### PR TITLE
{chem}[intel/2015a] ABINIT 7.11.6 + libxc 2.2.2 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
build doesn't work yet, configure currently fails at:

```
...
 ==============================================================================
 === Connectors / Fallbacks                                                 ===
 ==============================================================================

checking for the requested transferable I/O support... etsf_io+netcdf
checking whether to select a fallback for netcdf... no
checking netcdf.h usability... yes
checking netcdf.h presence... yes
checking for netcdf.h... yes
checking for Fortran module includes... none required
checking whether NetCDF Fortran wrappers work... no

        +--------------------------------------------------------------+
        | Connector detection failure                                  |
        +--------------------------------------------------------------+
...
```

@gmatteo: any idea what we're doing wrong?

This is our current configure command:

```
./configure
--prefix=/gpfs/scratch/projects/project_gpilot/vsc40023/software/ABINIT/7.11.6-intel-2015a
--with-mpi-prefix=$EBROOTIMPI/intel64
--with-trio-flavor='etsf_io+netcdf'
--with-dft=flavor='libxc'
--with-netcdf-incs="-I$EBROOTNETCDF/include -I$EBROOTNETCDFMINFORTRAN/include"
--with-netcdf-libs="-L$EBROOTNETCDF/lib -lnetcdf -L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff"
--with-libxc-incs="-I$EBROOTLIBXC/include"
--with-netcdf-libs="-L$EBROOTLIBXC/lib -lxc"
```

You told me to also pass this via a file specified via ` --with-config-file`, but that's not done here yet.

Can this be specified on the `configure` command line, rather than via a config file?

```
fcflags_opt_43_ptgroups='-O0 -g -extend-source'
```

File: easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '7.11.6'
```
Comment (boegel) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb]: current last public release is 7.10.4, so we should add another easyconfig for that?

http://www.abinit.org/downloads/source-packages/abinit-1

**Comment by gmatteo** [comment]
Hi,

--with-netcdf-libs="-L$EBROOTLIBXC/lib -lxc"
should be replaced with --with-libxc-libs

fcflags_opt_43_ptgroups='-O0 -g -extend-source'
is a trick I use to speed up the compilation because the compiler takes a lot of time to
compile the code in this directory. You can ignore this option in easybuild

File: easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c8b166ec8e65ad1d795d42b889fd772b']
+
+configopts = "--with-mpi-prefix=$EBROOTIMPI/intel64 --with-trio-flavor='etsf_io+netcdf' --with-dft=flavor='libxc' "
+configopts += '--with-netcdf-incs="-I$EBROOTNETCDF/include -I$EBROOTNETCDFMINFORTRAN/include" '
+configopts += '--with-netcdf-libs="-L$EBROOTNETCDF/lib -lnetcdf -L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff" '
+configopts += '--with-libxc-incs="-I$EBROOTLIBXC/include" --with-netcdf-libs="-L$EBROOTLIBXC/lib -lxc" '
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb]: typo here `netcdf` should be `libxc`

File: easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.11.6-intel-2015a.eb]: see #1650

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/18ff11464fc5fe9bbbda for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3ab0987bdcdc9532b986 for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the feedback @gmatteo, and @JensTimmerman for the review!

**Pull request merged**